service/assessment/assessment_test.go

```diff
@@ -238,53 +245,66 @@ func TestAssessEvidences(t *testing.T) {
 		UnimplementedAssessmentServer assessment.UnimplementedAssessmentServer
 	}
 	type args struct {
-		stream assessment.Assessment_AssessEvidencesServer
+		streamToServer *mockAssessmentServerStream
```

Comment:
In the arguments it would be better to use the interface `assessment.Assessment_AssessEvidencesServer` rather then the implementation